Bottom: 06cda31769fc0713237659ebbbfced12c9d692be
Top:    1efd1a63fce0a680ae423da3f98f3397659c6870
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-03 15:02:42 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/lib/mounts.c b/lib/mounts.c
index 494fc7b..550445c 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -46,8 +46,6 @@ static const char mnt_name_template[]      = "automount(pid%u)";
 static struct kernel_mod_version kver = {0, 0};
 static const char kver_options_template[]  = "fd=%d,pgrp=%u,minproto=3,maxproto=5";
 
-unsigned long mountflags = MS_UNBINDABLE|MS_MGC_VAL;
-
 unsigned int linux_version_code(void)
 {
 	struct utsname my_utsname;
